src/send/usePrepareSendTransactions.ts

```diff
@@ -68,10 +68,12 @@ export function usePrepareSendTransactions() {
   const [prepareTransactionsResult, setPrepareTransactionsResult] = useState<
     PreparedTransactionsResult | undefined
   >()
+  const [prepareTransactionError, setPrepareTransactionError] = useState<Error | undefined>()
```

Comment:
We don't need to store the error, it's already stored in the async callback hook.

We can return it like this: `prepareTransactions.error`

(same for the result by the way).